#! /bin/sh /usr/share/dpatch/dpatch-run
## log-retrieve.dpatch by Diego Iastrubni  <diego.iastrubni@xorcom.com>
##
## DP: backport from the trunk (2.2.x): retrieve_conf will send it's output to /var/log/

@DPATCH@
diff -urNad freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/footer.php /tmp/dpep.7rnVmA/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/footer.php
--- freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/footer.php	2006-06-18 15:42:17.000000000 +0300
+++ /tmp/dpep.7rnVmA/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/footer.php	2006-11-21 16:54:44.000000000 +0200
@@ -65,8 +65,8 @@
 	
 	//run retrieve script
 	$retrieve = $amp_conf['AMPBIN'].'/retrieve_conf';
-	exec($retrieve.'>/dev/null');
-	
+        exec($retrieve.'>& ' . $asterisk_conf['astlogdir'] . "/freepbx-retrieve.log" );
+
 	require_once('common/php-asmanager.php');
 	$astman = new AGI_AsteriskManager();
 	if ($res = $astman->connect("127.0.0.1", $amp_conf["AMPMGRUSER"] , $amp_conf["AMPMGRPASS"])) {
@@ -81,8 +81,8 @@
 		
 		//bounce op_server.pl
 		$wOpBounce = rtrim($_SERVER['SCRIPT_FILENAME'],$currentFile).'bounce_op.sh';
-		exec($wOpBounce.'>/dev/null');
-		
+                exec($wOpBounce.'>& ' . $asterisk_conf['astlogdir'] . "/freepbx-opbounce.log" );
+						
 		//store asterisk reloaded status
 		$sql = "UPDATE admin SET value = 'false' WHERE variable = 'need_reload'"; 
 		$result = $db->query($sql); 
